### ISSUE TITLE
minimum SDK 

### ISSUE THREAD

**Issue by salim-lachdhaf** [sumup/sumup-android-sdk#196]
I don't understand why the minimun SDk is forced to 23 ?

**Comment by vladaman** [comment]
yeah, we have same issue with our devices. many of them are integrated for years and we can't just replace them every 2 years when sumup decides to drop support. Even worst, as of today old SDKs have stopped working today

See https://github.com/sumup/sumup-android-sdk/issues/206

**Comment by leila-sumup** [comment]
Hello, thanks for raising this, and apologies for the slow response.

We understand this is painful, especially when you have devices deployed in the field that aren't easy to swap out.

The minimum SDK requirement exists because we need to ensure security, reliability, and maintainability across our payment stack. Android 6.0 (SDK 23) was released in 2015 and no longer receives security patches from Google. Supporting older, unpatched OS versions introduces risk we're not comfortable with for a payments product.

Since this issue was opened, our minimum target has moved to SDK 35. We expect this to continue trending upward over time as older Android versions fall out of Google's support window. We believe keeping our SDK aligned with actively supported Android versions is the right call for the security and stability of the platform.
